### DISCUSSION TITLE
Settings - set up static width / minimal width for recipe display

### DISCUSSION THREAD

**Discussion by Marxone** [shedaniel/RoughlyEnoughItems#1191] [Q & A] [answered]
Hi, 

I noticed REI getting more popular in recent packs but there is one problem I have settings. 

When there are recipes from mods that have static GUI size, REI recipe view keeps resizing (width) based on the mod GUI. I wonder if there is some settings to set up either minimal width or static width of the recipe view. The problem is also with recipe tabs - when the width changes, icons on recipe tabs move/realign. 

![REI](https://user-images.githubusercontent.com/56160142/200118529-5a05d7fb-d45f-497d-91e7-0fc947f6d6db.gif)

**Comment by shedaniel** [comment, marked as answer]
I have just pushed an update to a new logic of the tabs, see if that helps, static width is still in consideration however

**Reply by Marxone** [reply]
After testing updated version for a bit it seems like an improvement for the better. Scrolling through tabs is now smoother, without the hickups for changing amount of visible tabs. 
I'd say static width would be a nice to have, but this change will do :) You have my thanks. 


edit: tested on FTB Stoneblock 3, 1.18.2 pack
